corona-warn-app/cwa-app-android
# Material Design 🎨 (DEV)

- Move completely to MD Theme
-  `ProgressBar`  ->  `CircularProgressIndicator`
- Remove some redundant styles 


### Testing 
- Run the App and go through the screens 
- Check on older Android phones
- [Screenshots](https://app.circleci.com/pipelines/github/corona-warn-app/cwa-app-android/16202/workflows/d107eb6e-2f7c-4740-8298-aff20fb16aea/jobs/132021)

### TODO
- [x] Revert screenshots job
